helm/dex-app/templates/dex-k8s-authenticator/deployment-giantswarm.yaml

```diff
@@ -1,114 +0,0 @@
-{{- if eq (include "has-giantswarm-connector" .) "true" -}}
```

Comment:
we do not use dex-k8s-authenticator for giantswarm anymore and these templates are never applied so I'm deleting them.